tests/test_api_build.py

```diff
@@ -1180,21 +1177,13 @@ def test_failed_recipe_leaves_folders(testing_config, testing_workdir):
         api.build(m)
     assert os.path.isdir(m.config.build_folder), 'build folder was removed'
     assert os.listdir(m.config.build_folder), 'build folder has no files'
+
     # make sure that it does not leave lock files, though, as these cause permission errors on
     #    centralized installations
-    any_locks = False
-    locks_list = set()
-    locale.getpreferredencoding(False)
-    for lock in locks:
-        if os.path.isfile(lock.lock_file):
-            any_locks = True
-            dest_path = base64.b64decode(os.path.basename(lock.lock_file))
-            if hasattr(dest_path, "decode"):
-                dest_path = dest_path.decode(errors="backslashreplace")
-            locks_list.add((lock.lock_file, dest_path))
-    assert not any_locks, "\n".join(
+    locks = {lock.lock_file for lock in locks if os.path.isfile(lock.lock_file)}
```

Comment:
this makes sense.. with removing the old set up variables.. and decoding..

Reply:
I removed it because, as far as I could tell, the decoding didn't expose anything meaningful. I also think the decoding was gibberish, so perhaps it used to mean something in the past before conda adopted `filelock` for the locking mechanism?